treeherder/etl/bugzilla.py

```diff
@@ -59,6 +62,8 @@ def reopen_intermittent_bugs(minimum_failures_to_reopen=1):
 
         try:
             reopen_request(url, method="PUT", headers=headers, json=data)
+            # NOTE: this will only toggle 1 bug_job_map entry, not all (if there are retriggers)
+            BugJobMap.objects.filter(job_id=job_id, bug_id=bug_id).update(bug_open=True)
```

Comment:
If I remember correct there is an unhandled edge case here: If a bug was public but later restricted and Treeherder cannot access it anymore, it remembers the last state. In other words, a restricted bug which is remembered as in incomplete state will remain like that and always be requested for reopening until no classifications are stored anymore. This would yield more `bug_open=True` than expected.

Reply:
interesting.  I am not sure how to handle this.  My intended query is to see bug_open and get distinct bug_ids, this would reduce it to at most 1x/day for each bug in this state.  I assume this is a low number of bugs.